.kitchen.yml

```diff
@@ -1,62 +1,77 @@
 ---
 driver:
-  name: vagrant
+  name: docker
```

Comment:
I took a look at https://github.com/someara/kitchen-dokken.
It sounds good, but right now we use kitchen-docker in all our ansible-roles. So I'l write it on my todo-list to check out dokken.